Etabs_Adapter/Etabs_Adapter.csproj

```diff
@@ -418,8 +421,13 @@
       xcopy "$(TargetDir)ETABS2016.dll"  "C:\ProgramData\BHoM\Assemblies" /Y
 xcopy "$(TargetDir)ETABSv17.dll"  "C:\ProgramData\BHoM\Assemblies" /Y
 xcopy "$(TargetDir)CSiAPIv1.dll"  "C:\ProgramData\BHoM\Assemblies" /Y
+
 </PostBuildEvent>
   </PropertyGroup>
+  <PropertyGroup>
+    <PreBuildEvent>
+    </PreBuildEvent>
+  </PropertyGroup>
```

Comment:
Think this may have been added in error?

Reply:
Hi @peterjamesnugent.
Not sure what happened with the code. Removed the four lines above.
Updated code in the latest commit